src/main/java/org/cascadebot/cascadebot/commands/fun/RandomColorSubCommand.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+
+package org.cascadebot.cascadebot.commands.fun;
+
+import net.dv8tion.jda.api.entities.Member;
+
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ISubCommand;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.ColorUtils;
+import org.cascadebot.cascadebot.utils.RandomUtils;
+
+public class RandomColorSubCommand implements ISubCommand {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        context.reply(ColorUtils.getColor(RandomUtils.randomColor(), context));
+    }
+
+    @Override
+    public String command() { return "color"; }
```

Comment:
For commands we're using the uk spelling for the actual command so this should be `colour`.